### PR TITLE
Alert detail page

### PR DESCRIPTION
**Pull request by hortha**
### Context

Users of the TRS Console need to be able to view details of alerts.

### Changes proposed in this pull request

Populate the /alerts/{alertId} page following the designs within the ‘Closed alert’ section ofthe Figma design (note that active alerts should be visible here too).

If no alert exists with the specified alertId return a 404.

The back link should go back to /persons/{personId}/alerts where {personId} is the ID of the person linked to the current alert.

### Guidance to review

CRM queries + UI + tests
Excluding detaillink functionality at the moment until CRM changes are done.

### Checklist

-   [x] Attach to Trello card
-   [ ] Rebased master
-   [ ] Cleaned commit history
-   [x] Tested by running locally

### CODE REVIEW

**Comment by hortha** [comment]
[Alert detail page](https://trello.com/c/XO7Q9hHb/1242-alert-detail-page)

File: TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/Alert/Index.cshtml [hunk 1]
```diff
@@ -1,7 +1,65 @@
...
+
+<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+        <h3 class="govuk-heading-s">Details</h3>
+        @if (!string.IsNullOrEmpty(@Model.Alert.Details))
+        {
+            <p class="govuk-body" data-testid="alert-details">
+            @{var lines = @Model.Alert.Details.Split(new string[] { "\r\n", "\r", "\n" }, StringSplitOptions.None);
```
Comment (gunndabad) [inline TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/Alert/Index.cshtml:42]: Can we move this into a tag helper to remove the duplication with `/persons/{id}/alerts`?

File: TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/Alert/Index.cshtml.cs [hunk 2]
```diff
@@ -1,10 +1,74 @@
...
+    [FromRoute]
+    public Guid AlertId { get; set; }
+
+    public AlertInfo? Alert { get; set; }
+
+    public bool IsActive { get; set; }
+
+    public Guid? PersonId { get; set; }
+
+    public async Task<IActionResult> OnPost()
```
Comment (gunndabad) [inline TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/Alert/Index.cshtml.cs:32]: I think we should have two separate, named handler methods for activate and reactivate.

File: TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/Alert/Index.cshtml (hunk 1 above)
Comment (hortha) [reply TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/Alert/Index.cshtml]: Done

File: TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/Alert/Index.cshtml.cs (hunk 2 above)
Comment (hortha) [reply TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Alerts/Alert/Index.cshtml.cs]: Done

**Review by gunndabad — APPROVED** [review]

**Pull request merged**